tf_patches/patch.diff

```diff
@@ -32,3 +32,15 @@ index b70ea8af5df..b54a895e306 100644
    } else if (tool_name == "tool_names") {
      return GetAvailableToolNames(session_snapshot);
    } else {
+diff --git a/tensorflow/compiler/xla/pjrt/pjrt_c_api_client.cc b/tensorflow/compiler/xla/pjrt/pjrt_c_api_client.cc
+index e6fda37012b..5d10dc157e2 100644
+--- a/tensorflow/compiler/xla/pjrt/pjrt_c_api_client.cc
++++ b/tensorflow/compiler/xla/pjrt/pjrt_c_api_client.cc
```

Comment:
we should be able to remove this once we update tf again right?

Reply:
Yeah, this should get removed next time we update the pin.